### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by kssasha**

### CODE REVIEW

File: Lesson_9/index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        <p>
+          Auto mechanic & Senior master at
+          <a href="https://www.autoservis-garant.cz/">Autoservis-Garant</a>
+        </p>
+      </td>
+    </tr>
+  </table>
+
+  <h2>
+    <u class="bottom_skils_right">Skils:</u>
```
Comment (RomanIlkiv) [inline Lesson_9/index.html:84]: try to use one convention of naming classes through all html file
I mean you have classes like that: `profile-photo` or `bottom_skils_right`
use dashes in both or underscores in both

File: Lesson_9/index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      <span class="position">Auto mechanic</span>
+      <span class="position">Junior Front-End Developer in Future</span>
+    </div>
+    <div>
+      <img src="./img/my_photo.jpg" width="125" height="125" alt="My photo" class="profile-photo"/>
+    </div>
+  </div>
+
+  <h2>
+    <u class="bottom">Work Experience</u>
```
Comment (RomanIlkiv) [inline Lesson_9/index.html:26]: better to use class `work-experience` or something like that

File: Lesson_9/index.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    </tr>
+  </table>
+
+  <h2>
+    <u class="bottom_skils_right">Skils:</u>
+  </h2>
+
+  <ul>
+    <li>
+      <h3>Auto mechanic</h3>
```
Comment (RomanIlkiv) [inline Lesson_9/index.html:89]: why h3 inside of li?
you can change font-size already in css and not use redundant tags

**Comment by kssasha** [comment]
thank you for the advice, I will remember this in the future

30 мая 2023, 09:00:36, от "Roman" ***@***.***>:


@RomanIlkiv commented on this pull request.
In Lesson_9/index.html:
> +      </td>
+    </tr>
+    
+    <tr>
+      <td>2022-Present</td>
+      <td>
+        <p>
+          Auto mechanic & Senior master at
+          <a href="https://www.autoservis-garant.cz/">Autoservis-Garant</a>
+        </p>
+      </td>
+    </tr>
+  </table>
+
+  <h2>
+    <u class="bottom_skils_right">Skils:</u>
try to use one convention of naming classes through all html file
I mean you have classes like that: profile-photo or bottom_skils_right
use dashes in both or underscores in both
In Lesson_9/index.html:
> +</head>
+
+<body>
+  <div class="header">
+    <div>
+      <h1>Oleksandr Kalenikov</h1>
+      <span class="position">Auto mechanic</span>
+      <span class="position">Junior Front-End Developer in Future</span>
+    </div>
+    <div>
+      My photo
+    </div>
+  </div>
+
+  <h2>
+    <u class="bottom">Work Experience</u>
better to use class work-experience or something like that
In Lesson_9/index.html:
> +      <td>
+        <p>
+          Auto mechanic & Senior master at
+          <a href="https://www.autoservis-garant.cz/">Autoservis-Garant</a>
+        </p>
+      </td>
+    </tr>
+  </table>
+
+  <h2>
+    <u class="bottom_skils_right">Skils:</u>
+  </h2>
+
+  <ul>
+    <li>
+      <h3>Auto mechanic</h3>
why h3 inside of li?
you can change font-size already in css and not use redundant tags
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>


--
С уважением Саша!

**Pull request merged**